resources/core/templates/tests/test-external-solution.yaml

```diff
@@ -4,11 +4,11 @@
 apiVersion: "testing.kyma-project.io/v1alpha1"
 kind: TestDefinition
 metadata:
-  name: {{ $.Chart.Name }}-test-external-solution-{{ $suffix | replace "_" "-" }}
+  name: {{ $.Chart.Name }}-test-external-solution
```

Comment:
Can we also remove the for loop above? We only have one test at the moment. The loop is only used to get `$val.testsuite` which evaluates to `e2e-event-mesh`. So we still use the name inside the TestDefiniton.

Reply:
Done